test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Indexers/FunctionIndexerTests.cs

```diff
@@ -114,7 +114,7 @@ public async Task IndexMethod_IfMethodReturnsAsyncVoid_Throws()
             // Act & Assert
             await product.IndexMethodAsync(typeof(FunctionIndexerTests).GetMethod("ReturnAsyncVoid"), index, CancellationToken.None);
 
-            string expectedMessage = "Function 'ReturnAsyncVoid' is async but does not return a Task. Your function may not run correctly.";
+            string expectedMessage = "Function 'FunctionIndexerTests.ReturnAsyncVoid' is async but does not return a Task. Your function may not run correctly.";
```

Comment:
This is an okay log change -- it's more descriptive and it should have been like this to begin with.